spec/models/restaurant_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+RSpec.describe Restaurant, type: :model do
+  context 'validations' do
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_num_in_range(:rating, 0, 3) }
+    it { is_expected.to validate_num_in_range(:max_delivery_time_min, 1, 720) }
+    it { is_expected.to validate_num_in_range(:latitude, -90, 90) }
+    it { is_expected.to validate_num_in_range(:longitude, -180, 180) }
+    it { is_expected.to validate_inclusion_of(:cuisine).in_array(Restaurant::E_CUISINES) }
+  end
+
+  context 'defaults' do
+    let(:restaurant) { Restaurant.create(name: 'test', cuisine: Restaurant::E_CUISINES[0]) }
```

Comment:
In tests we prefer as much as randomability as possible. Take a look at the [sample](https://ruby-doc.org/core-2.5.1/Array.html#method-i-sample) method